code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -418,6 +418,15 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 	var/obj/item/organ/external/hand/l_hand = get_limb_by_name("l_hand")
 	var/obj/item/organ/external/hand/right/r_hand = get_limb_by_name("r_hand")
 
+	if(isgolem(src))
+		var/datum/species/golem/G = src.dna.species
```

Comment:
```suggestion
		var/datum/species/golem/G = dna.species
```
remember that you don't need `src` to refer to variables and methods on the object you're currently scoped to, src is already implied
`src.proc` is the same thing as `proc` in almost every case (except constructors but this isn't one of those)